### PR TITLE
Tidy up.

### PR DESCRIPTION
**Pull request by ben-eb**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,8 +20,15 @@
   ],
   "license": "MIT",
   "devDependencies": {
-    "tap-spec": "^4.1.0",
-    "tape": "^4.2.0"
+    "ava": "^0.11.0",
+    "babel-cli": "^6.4.0",
+    "babel-core": "^6.4.0",
+    "babel-plugin-add-module-exports": "^0.1.2",
+    "babel-preset-es2015": "^6.3.13",
```
Comment (TrySound) [inline package.json]: Why we need both?

File: .eslintrc [hunk 2]
```diff
@@ -0,0 +1,72 @@
+root: true
```
Comment (TrySound) [inline .eslintrc]: God, let's make shared config maybe by your name or for cssnano (which I can follow too in my cssnano projects). I more and more don't like custom lint configs :)

File: src/index.js [hunk 3]
```diff
@@ -1,21 +1,17 @@
...
     return unit(node.value);
 }
 
 function transformAtRule (css, atRuleRegex, propRegex) {
-    var cache = {};
-    var ruleCache = [];
-    var declCache = [];
+    let cache = {};
+    let ruleCache = [];
+    let declCache = [];
```
Comment (TrySound) [inline src/index.js]: All objects can be constant because references are not changed. only properties.

File: package.json (hunk 1 above)
Comment (ben-eb) [inline package.json]: preset-es2015-loose extends from preset-es2015.

**Pull request merged**